src/Indice.Services/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -124,7 +124,27 @@ public static IServiceCollection AddSmsServiceApifon(this IServiceCollection ser
                                         .ConfigureHttpClient(httpClient => {
                                             httpClient.BaseAddress = new Uri("https://ars.apifon.com/services/api/v1/sms/");
                                         });
-        if (options.ConfigurePrimaryHttpMessageHandler != null) {
+        if (options.ConfigurePrimaryHttpMessageHandler is not null) {
+            httpClientBuilder.ConfigurePrimaryHttpMessageHandler(options.ConfigurePrimaryHttpMessageHandler);
+        }
+        return services;
+    }
+
+    /// <summary>Adds an instance of <see cref="ISmsService"/> using Apifon IM service gateway.</summary>
+    /// <param name="services">Specifies the contract for a collection of service descriptors.</param>
+    /// <param name="configuration">Represents a set of key/value application configuration properties.</param>
+    /// <param name="configure">Configure the available options. Null to use defaults.</param>
+    public static IServiceCollection AddViberServiceApifon(this IServiceCollection services, IConfiguration configuration, Action<SmsServiceApifonOptions> configure = null) {
```

Comment:
This is not an Impementation of IViberService this does not exist. You must follow the naming convention so that it is discoverable along side the other ISmsService implementations. 
rename `AddViberServiceApifon` to `AddSmsServiceApifonIM`